### DISCUSSION TITLE
Port Forwarding via CLI?

### DISCUSSION THREAD

**Discussion by nickmitricka** [rancher-sandbox/rancher-desktop#3025] [Q&A] [answered]
I'm new to Rancher-Desktop and trying to understand capabilities. I know there is the GUI way to setup port forwarding, but are there any command line tools (rdctl or limactl) that will allow me to update the port forwarding configuration?

**Comment by Rocketchat23** [comment, marked as answer]
There is a configuration for that

**Reply by jandubois** [reply]
There is no CLI mechanism to modify the port-forwarding of cluster-internal ports right now.

Note that all NodePorts are automatically forwarded to the host already, so configuring this for ingress should not be necessary; the UI is only needed if you want to e.g. access a service that is otherwise not exposed outside the cluster.

**Reply by nickmitricka** [reply]
@jandubois  -  Yes, the item is not currently exposed outside of the cluster.  Is the CLI mechanism on the roadmap?

**Reply by jandubois** [reply]
Not explicitly, afaict, so please file an enhancement request for it!

**Comment by segevfiner** [comment]
Did you open an issue for this?

**Comment by jandubois** [comment]
There is #4171 now, which partially covers it. Once there is an API, it would be accessible via `rdctl api ...`. As a second step there should probably be a user-level command for it too, but that too will require the API to be in place first.
